### PR TITLE
State target gen progress

### LINKED ISSUE #28: Add Data to State Details Page
Blocked by issue #20, which makes the page.

This should include fleshing out all of the graphs and all of the missing numerical data (indicating by a ?).

### PR DESCRIPTION
**Pull request by swatland610**
This PR should ideally close #28. 

Here's what this PR entails: 
1. Updates to the `state-details.js` page that pulls in data from the graphql queries (similarly to how Dylan set up buildings and vehicles)
2. a new file in `components` called `WindSolarChart.js` that details progress towards our target generation numbers for each state

One language & data choice that I made. I opted to keep the data centered on **installed capacity** rather than **number of solar or wind farms**. I have two reasons for this, I'll summarize them quick...

1. This is how experts talk about this. I couldn't find anyone else who was trying to accurately calculate even a ballpark number for solar panels & wind turbines needed. Plus with assumed improvements to technology over time, it feels like too much of a moving target. 
2. This makes our math SOOOO much easier - and I feel stronger about the results. 

Here's a screenshot of the addition: 
Screen Shot 2022-05-18 at 12 44 58 AM

### CODE REVIEW

**Comment by swatland610** [comment]
@derekeder @nofurtherinformation @vkoves 

this fixes my last PR with all of the merge conflicts - just a heads up!

File: data/raw/state_renewable_gen_targets.csv [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-UT,32313.11,20737.31,20367,34880.49,34866.06,3670.87,73417.42
-VT,42.74,5160.44,5068.29,4280.87,5477.02,513.57,10271.46
-VA,66616.81,59249.45,39420,80264.05,76757.91,8264.31,165286.26
-WA,19495.65,45297.16,42840.57,47631.4,54620.31,5381.67,107633.38
-WV,53182.04,12561.31,8812.14,38626.24,32201.48,3727.77,74555.49
-WI,45868.54,40773.82,46720,60734.9,65959.34,6668.12,133362.36
-WY,35168.22,6434.62,4213.14,24061.24,19463.94,2290.8,45815.98
\ No newline at end of file
+state,rnw_generation_fossil_fuels,rnw_generation_transportation,rnw_generation_buildings,total_gen_by_solar,total_gen_by_wind,total_gen_by_other,total_rnw_gen_needed,current_solar,current_wind,perc_solar_target,perc_wind_target
+AL,76967.78,40986.18,14089,68819.6,56621.21,"6,602","132,043",383.58,0,1,0
```
Comment (nofurtherinformation) [inline data/raw/state_renewable_gen_targets.csv:2]: @swatland610 Alaska is missing here, which is currently causing the build to fail due to missing data

**Comment by nofurtherinformation** [comment]
Hey @swatland610 - just took a look at the tests are failing due to Alaska and Hawaii missing in the data. 

Plugging in dummy data, it seems to be fine, so the rest of the PR is good to go. I'd suggest putting in a placeholder rows for them and doing a conditional check to see if the data exists to render the new parts of the state details component!

File: data/raw/state_renewable_gen_targets.csv [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
+AZ,64565.23,47611.93,14766.86,65832.16,54764.66,"6,347","126,944",8749.49,644.16,13,1
```
Comment (nofurtherinformation) [inline data/raw/state_renewable_gen_targets.csv:3]: Adding Hawaii, too, should fix the build issue.

**Review by nofurtherinformation — CHANGES_REQUESTED** [review]

**Comment by nofurtherinformation** [comment]
Thanks @swatland610 ! I just did a quick lint run to fix the formatting issues our linter picked up, and we should be good to go!

**Pull request merged**